addons/mrp/models/mrp_production.py

```diff
@@ -52,6 +52,7 @@ def _get_default_is_locked(self):
         'product.product', 'Product',
         domain="""[
             ('type', 'in', ['product', 'consu']),
+            ('id', 'not in', disabled_product_ids),
```

Comment:
We don't base the domain on dynamic computed field due to performance issue. We aldready removed that kind of domain in the past https://github.com/odoo/odoo/commit/0468e3ccc11042236a3b4bd1c7164a7b780bc481